WISE-Community/WISE-Client
# feat(Authoring): Add Dynamic Prompt authoring

## Changes

Added authoring for Dynamic Prompt.

## Test

- If the Dynamic Prompt is enabled, make sure the regular prompt authoring becomes hidden
- Make sure the reference component step authoring works
- Make sure the reference component component authoring works
- Make sure the Dynamic Prompt peer grouping tag authoring works
- Make sure the pre prompt authoring works
- Make sure the prompt rules authoring works
- Make sure the post prompt authoring works

#840
